ts/components/leftpane/overlay/OverlayMessage.tsx

```diff
@@ -89,16 +107,31 @@ export const OverlayMessage = () => {
         placeholder={placeholder}
         onChange={setPubkeyOrOns}
         dataTestId="new-session-conversation"
+        onPressEnter={handleMessageButtonClick}
       />
 
       <SessionSpinner loading={loading} />
 
-      <div className="session-description-long">{descriptionLong}</div>
+      <Flex container={true} width="100%">
+        <SpacerMD />
+        <YourSessionIDPill />
+        <SpacerMD />
+      </Flex>
+      <Flex
+        container={true}
+        justifyContent="space-between"
+        alignItems="center"
+        width="100%"
+        padding="0 var(--margins-md)  " // YourSessionIDSelectable already has a left margin of 15px
```

Comment:
remove the extra spaces after `var(--margins-md)`

Reply:
updated